provision/roles/aws/templates/output_formatter.j2

```diff
@@ -1 +1 @@
-{{ topology_outputs | to_nice_yaml }}
+{{ topology_outputs }}
```

Comment:
As discussed, this will need to be returned. I am aware of the other PR with ansible v2.2.0 in place to ensure inventories are properly generated. AIUI, the to_nice_yaml filter is also updated.